WalletWasabi.Gui/Controls/WalletExplorer/SendTabViewModel.cs

```diff
@@ -239,7 +239,11 @@ public SendTabViewModel(WalletViewModel walletViewModel, bool isTransactionBuild
 					}
 				});
 
-			FeeRateCommand = ReactiveCommand.Create(ChangeFeeRateDisplay, outputScheduler: RxApp.MainThreadScheduler);
+			FeeRateCommand = ReactiveCommand.CreateFromTask(async () =>
+			{
+				ChangeFeeRateDisplay();
+				await Global.UiConfig.ToFileAsync();
```

Comment:
Should catch exception here I think.

Reply:
Or more better, the tofile shouldn't happen here, but on the subscription later.